pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -543,6 +543,78 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryBasicUserRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("imagestreams", "images"),
+				},
+				{
+					Verbs:     sets.NewString("create", "list"),
+					Resources: sets.NewString("projectrequests"),
+				},
+				{
+					Verbs:     sets.NewString("list"),
+					Resources: sets.NewString("projects", "subjectaccessreview", "localsubjectaccessreview"),
+				},
+			},
+		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryAdminRoleName,
```

Comment:
I think you want
 1.  "get, list, watch, create, update, patch, delete" on "imagestreams, imagestreamtags, imagestreammapping, imagestreams/secrets, imagestreamimages, imagestreamimports"
 1. "get, update" on "imagestreams/layers"
 1. "get list watch create update patch delete" on "roles, rolebindings, resourceaccessreviews subjectaccessreviews localresourceaccessreviews localsubjectaccessreviews"
 1. get list watch on policies policybindings

that should give you power to manipulate the registry and control access to your repo.